lib/general.js

```diff
@@ -44,13 +45,29 @@ class NodeVersionCheck extends DoctorCheck {
 }
 checks.push(new NodeVersionCheck());
 
+
+async function getNpmCmdPath () {
```

Comment:
getNpmBinaryName

Reply:
probably, this function is not even needed, since it only would contain one operator